src/Razor/src/Microsoft.VisualStudio.RazorExtension/EmbeddedGrammars/aspnetcorerazor.tmLanguage.tmTheme

```diff
@@ -205,13 +205,27 @@
         </dict>
       </dict>
 
+      <dict>
```

Comment:
Big thanks to Bhavya who did all the work of figuring out which scopes were missing and what vsclassificationType they should map to.